networkx/algorithms/components/weakly_connected.py

```diff
@@ -123,6 +131,18 @@ def is_weakly_connected(G):
     NetworkXNotImplemented
         If G is undirected.
 
+    Examples
+    --------
+    >>> import networkx as nx
+    >>> G = nx.DiGraph()
+    >>> G.add_nodes_from([0, 1, 2, 3])
+    >>> G.add_edges_from([(0, 1), (2, 1)])
```

Comment:
Similar comment as above, though in this case accounting for the addition of the unconnected node:

```suggestion
    >>> G = nx.DiGraph([(0, 1), (2, 1)])
    >>> G.add_node(3)
```

Reply:
I have made the changes.